communication-prefs/communication-prefs.js

```diff
@@ -12,7 +12,19 @@ module.exports = {
     var AVAILABLE_LANGUAGES = [
       'de',
       'en',
+      'en-au',
+      'en-bz',
+      'en-ca',
+      'en-cb',
+      'en-gb',
+      'en-ie',
+      'en-in',
+      'en-jm',
+      'en-nz',
+      'en-ph',
+      'en-tt',
       'en-us',
+      'en-za',
```

Comment:
are we missing en-zw ?

![image](https://cloud.githubusercontent.com/assets/128755/11123811/da927780-892f-11e5-89f0-acfdcec0a17e.png)